src/main/java/org/inferred/freebuilder/processor/CodeGenerator.java

```diff
@@ -197,12 +201,12 @@ private static void addMergeFromBuilderMethod(SourceBuilder code, Metadata metad
         .addLine(" * Does not affect any properties not set on the input.")
         .addLine(" */")
         .addLine("public %1$s mergeFrom(%1$s template) {", metadata.getBuilder());
-    Block body = new Block(code);
+    Block body = Block.methodBody(code, "template");
```

Comment:
worth pulling out these magic strings and above (template, value)

Reply:
As discussed offline, I think in this case this reduces readability (%s in the addLines above) without improving refactorability (only used in this method)